apps/wallet/src/ui/app/pages/dapp-tx-approval/index.tsx

```diff
@@ -284,24 +316,29 @@ export function DappTxApprovalPage() {
                                         Transaction Type
                                     </div>
                                     <div className="font-semibold text-sui-steel-darker">
-                                        {txRequest?.unSerializedTxn?.kind ??
-                                            txRequest?.tx?.type}
+                                        {valuesContent[0].content}
                                     </div>
                                 </>
                             }
                         >
                             <div className={st.content}>
-                                {valuesContent.map(
-                                    ({ label, content, loading = false }) => (
-                                        <div key={label} className={st.row}>
-                                            <TransactionTypeCard
-                                                label={label}
-                                                content={content}
-                                                loading={loading}
-                                            />
-                                        </div>
-                                    )
-                                )}
+                                {valuesContent
+                                    .slice(1)
```

Comment:
I'm curious why this slice is being done?

Reply:
just removes the `'Transaction Type:  ...`   item since we display that as the title.